source/sfp/4_vegetation/sfp_407_nl_bhm-opnameschaal/12_appendices.Rmd

```diff
@@ -0,0 +1,73 @@
+# (APPENDIX) Bijlagen {.unnumbered}
+
+# Bijlage Visuele ondersteuning beheermonitoringschaal {#bijlage-visueel}
+
+De figuren geven voor elke code uit de beheermonitoringschaal een minimale en maximale visuele indruk van de waargenomen aantallen die men waarneemtbij het doorkruisen van een inventarisatie-eenheid (Figuur \@ref(fig:visueleondersteuning)).
+ZS = zeer schaars, S = schaars, WT = weinig talrijk, T = talrijk, B = bedekkend, KB = kwartbedekkend, HB = halfbedekkend en D = dominant.
+
+(ref:visueleondersteuning) Visuele ondersteuning van de codes van de beheermonitoringschaal. ZS = zeer schaars, S = schaars, WT = weinig talrijk, T = talrijk, B = bedekkend, KB = kwartbedekkend, HB = halfbedekkend en D = dominant.
+
+```{r visueleondersteuning, fig.cap="(ref:visueleondersteuning)", fig.show=ifelse(is_html_output(), "hold", "asis"), fig.align="center"}
```

Comment:
Je krijgt een caption bij elk van de acht figuren en dat ziet er zo wat raar uit. En de verwijzing vanuit de tekst naar de figuur werkt niet. Suggestie: de eerste 7 figuren eerst toevoegen zonder caption. In een andere chunk de laatste figuur toevoegen met caption. Ofwel één chunk per figuur met aangepaste caption.

Reply:
Begrijp niet helemaal wat je bedoelt, ik krijg maar één caption onderaan alle figuren als ik render? De verwijzing werkt inderdaad niet. Heb daar je suggestie gebrobeerd maar ik krijg Latex error. Kan jij eens bekijken @hansvancalster ??

Reply:
Ik heb enkel de gerenderde pdf bekeken. Daar krijg je dus een caption per figuur.

Reply:
Opgelost in [a9fa4b0](https://github.com/inbo/protocolsource/pull/105/commits/a9fa4b0660f2148a1e5b1c36497a030ce86bffbe)

Reply:
Was ik toch ni rap zelf opgekomen op dit {r fig.show=ifelse(is_html_output(), "hold", "asis")}
merci!